eng/Version.Details.xml

```diff
@@ -50,9 +50,9 @@
       <Uri>https://github.com/microsoft/vstest</Uri>
       <Sha>55e7e45431c9c05656c999b902686e7402664573</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Sdk.Web" Version="3.1.100-preview2.19522.3">
+    <Dependency Name="Microsoft.NET.Sdk.Web" Version="5.0.100-alpha1.19523.3">
```

Comment:
@nguerrera this looks wrong for the 3.1 branch. (@lewing pointed this out to me)

Reply:
@mmitche @livarcocc @dsplaisted @wtgodbe Any idea what happened here. A subscription must be wrong somewhere

Reply:
Looking

Reply:
It's not a bad sub, it's this: https://github.com/aspnet/websdk/commit/ff03fba04896ceb14d82d1ed45ef091332f4f326#diff-8b8f08ffbf7b863fb3700c1718eeb4cbL23-R24. @vijayrkn why did we merge all of those branding changes from master back into 3.1?

Reply:
The dependency versions in websdk also should've stayed at 3.1

Reply:
I think I see what happened, a PR was created from release/3.1.1xx to master, then the conflicts were resolved thereby updating release/3.1.1xx to master. This has happened before and the lesson is never, ever, ever create a PR between release branches, always use an intermediate branch for the PR where conflicts can be resolved.

Reply:
aspnet/websdk#1001

Reply:
My bad! I will rollback that commit in release/3.1.1 branch.

Reply:
I see - would it be better to do a `git reset --hard HEAD~1` in websdk release/3.1, or put up a PR reverting the change, for visibility?

Reply:
It is better to do a force rewind IMHO. If you revert, then the revert will merge to master cleanly and revert master to 3.1. People disagree on this though.

Reply:
I sent #3306 to revert this update